itou/www/siae_evaluations_views/tests/tests_siaes_views.py

```diff
@@ -337,6 +442,27 @@ def test_post(self):
             == EvaluatedAdministrativeCriteria.objects.first().administrative_criteria
         )
 
+    def test_post_with_submission_freezed_at(self):
+        evaluated_job_application = create_evaluated_siae_with_consistent_datas(self.siae, self.user)
+        criterion = (
+            evaluated_job_application.job_application.eligibility_diagnosis.selectedadministrativecriteria_set.first()
+        )
+        # Freeze submission
+        evaluated_job_application.evaluated_siae.submission_freezed_at = timezone.now()
+        evaluated_job_application.evaluated_siae.save(update_fields=("submission_freezed_at",))
+
+        url = reverse(
+            "siae_evaluations_views:siae_select_criteria",
+            kwargs={"evaluated_job_application_pk": evaluated_job_application.pk},
+        )
+
+        self.client.force_login(self.user)
+
+        post_data = {criterion.administrative_criteria.key: True}
```

Comment:
Un navigateur ne peut pas envoyer de booléen.
```suggestion
        post_data = {criterion.administrative_criteria.key: "on"}
```